enctests/testframework/encoders/ffmpeg_encoder.py

```diff
@@ -48,7 +48,7 @@ def run_wedges(self) -> dict:
         for wedge_name, wedge in self.test_config.get('wedges', {}).items():
             # Test name is based on main test name and wedge name
             test_name = f"{self.test_config.get('name')}-{wedge_name}"
-            out_file = self.get_output_filename(test_name)
+            (out_file, testbasename) = self.get_output_filename(test_name)
```

Comment:
```suggestion
            out_file, testbasename = self.get_output_filename(test_name)
```
There's no need to wrap this in parentheses.